src/fieri/.env

```diff
@@ -1,4 +1,5 @@
 FIERI_FOODCRITIC_FAIL_TAGS=any
 FIERI_FOODCRITIC_TAGS=~FC031 ~FC045
-FIERI_RESULTS_ENDPOINT=http://supermarket.getchef.com/api/v1/cookbook-verisons/evaluation
+FIERI_FOODCRITIC_ENDPOINT=http://supermarket.getchef.com/api/v1/cookbook-verisons/foodcritic_evaluation
+FIERI_COLLABORATORS_ENDPOINT=http://localhost:3000/api/v1/cookbook-versions/collaborators_evaluation
```

Comment:
Upon pondering this some more, I foresee pain in adding a new endpoint ENV var for each metric type. Most of the URL is the same, but would have to be repeated in N settings. As new metrics are added to the system, using them would require installers to add more ENV vars to their configs.

An option:

* Go back to `FIERI_RESULTS_ENDPOINT` and have it default to `http://localhost:3000/api/v1/cookbook-versions`.
* Fieri would append the metric endpoint itself—`foodcritic_evaluation` or `collaborators_evaluation`—to that results endpoint.

This way, an installation of Supermarket with Fieri enabled would only have to provide the URL to their custom domain + `api/v1/cookbook-versions` once.